config/database.yml

```diff
@@ -13,5 +13,5 @@ development:
   database: c2_development
 test:
   <<: *default
-  database: c2_test
+  url: postgresql://localhost/c2_test
```

Comment:
yes very intentional.

right now, if you have a DATABASE_URL defined in your local .env file, it will override the test db name and nuke your dev db while tests are running. Not good.